### PR TITLE
Adds stride to tile_conv1d

### PR DESCRIPTION
**Pull request by save-buffer**
v3 appears to maintain an IPC of 0.9781, which is good.

### CODE REVIEW

**Comment by save-buffer** [comment]
Sorry, I messed up. This doesn't pass correctness yet! Please don't merge until I fix.

**Comment by save-buffer** [comment]
OK new status: After optimizing heavily with strides, I managed to get it down to 1098 cycles with an IPC of 0.89. There's still a little bit of room for optimization because compiler is generating a `mul` instruction that I've been trying to get rid of with limited success. 
The kernels also now generate correct results!

**Comment by save-buffer** [comment]
After some more optimization, I think the limit is `v7`, which does `0.9808` IPC and `939` cycles.

File: examples/tile_conv1d/Makefile [hunk 1]
```diff
@@ -69,7 +69,7 @@ KERNEL_INCLUDES     += -I$(CURRENT_PATH)/kernel/include
 
 # Define the default kernel.cpp file. If KERNEL_DEFAULT is not defined it will
 # be set to kernel.cpp in the same directory as this Makefile.
-DEFAULT_VERSION     := v0
+DEFAULT_VERSION     := v7
```
Comment (drichmond) [inline examples/tile_conv1d/Makefile]: Let's keep the default v0

**Comment by drichmond** [comment]
Cool, can you update the descriptions in the readme?

**Comment by save-buffer** [comment]
Sorry didn't see that you left comments! Should be updated now.

**Comment by bornaehsani** [comment]
Looks good to me! Great work!

**Pull request merged**